waggle-dance/pom.xml

```diff
@@ -25,6 +26,17 @@
       <version>${project.version}</version>
       <classifier>exec</classifier>
     </dependency>
+    <dependency>
+      <groupId>com.hotels</groupId>
+      <artifactId>waggle-dance-rest</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
```

Comment:
Why do we need the exclusion? Isn't everything here at the same version?

Reply:
classifier is different

Reply:
Hmm, OK, so even though we specifically depend on the `exec` classifier of core above this is then pulling in another version of it transitively?

Reply:
yeah it's a different artifact as far as maven is concerned.